tests/test_package.py

```diff
@@ -0,0 +1,66 @@
+import io
+import logging
+import os
+import shutil
+from contextlib import contextmanager
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from snowcli.cli.snowpark import package
+
+from tests.test_data import test_data
+
+
+class TestPackage:
```

Comment:
Tests for `package_upload` are missing

Reply:
Yes, `package_upload` return type is None, and it`s purpose is in side-effect of uploading the zipped package to Snowflake.
Of course, i could mock every stage, but the test would be really checking if mocks return correct values - therefore i think it`s rather a candidate for an. integration test